README.md

```diff
@@ -177,6 +177,8 @@ API documentation is available [here](https://jellyfish-dev.github.io/react-nati
 
 ## Usage
 
+Important note!! Since version 0.2.0 call function initializeWebRTC() once in your app before using any other functionality.
```

Comment:
This is a weird note. Write a short example like here: https://github.com/jellyfish-dev/react-native-membrane-webrtc?tab=readme-ov-file#usage on how to use the library with initializeWebRTC included and/or add it in the changelog when releasing a new version. We also have examples here: https://jellyfish-dev.github.io/jellyfish-docs/tutorials/react-native

Most likely user is going to look for a tutorial/example when starting developing and then looking for changelog when updating the library. Nobody is going to look here when creating a new app or updating an exisiting app.